### DISCUSSION TITLE
A way to save the same PC with a Tailscale IP and LAN IP.

### DISCUSSION THREAD

**Discussion by CorporallClegg** [ClassicOldSong/moonlight-android#68] [Q&A]
Is there a known way to save a PC twice in Artemis? I would like to have a the same PC with my Tailscale IP and the local LAN IP listed as two separate PC's. Currently Artemis will not allow me to add two instances of the same PC even if they have different IP's is there a way to change the PC name in Artemis to achieve this or can/should that be done in Apollo?

**Comment by ClassicOldSong** [comment]
Cure there's no way to do this due to how moonlight was implemented. You can use some app cloning methods and add a separate IP in another Artemis instance.

**Comment by parkerlreed** [comment]
Why not only have the Tailscale IP? If it's local it won't route through your Internet.

```
Same WiFi
[parker@parker-framework ~]$ traceroute 100.111.199.55
traceroute to 100.111.199.55 (100.111.199.55), 30 hops max, 60 byte packets
 1  thinknas.tailfb618d.ts.net (100.111.199.55)  5.732 ms * *

Different WiFi
[parker@parker-framework ~]$ traceroute 100.111.199.55
traceroute to 100.111.199.55 (100.111.199.55), 30 hops max, 60 byte packets
 1  thinknas.tailfb618d.ts.net (100.111.199.55)  61.621 ms  61.614 ms  61.615 ms
```
